arch/x86/Kconfig

```diff
@@ -138,6 +138,7 @@ config X86
 	select HAVE_ARCH_JUMP_LABEL_RELATIVE
 	select HAVE_ARCH_KASAN			if X86_64
 	select HAVE_ARCH_KASAN_VMALLOC		if X86_64
+	select HAVE_ARCH_KFENCE
```

Comment:
Hm, wonder how this worked before.

Reply:
config KFENCE just didn't depend on this. I've added it together with this.